client/src/components/ResourceMap.jsx

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import React, { useState, useEffect } from 'react';
+import '../css/ResourceMap.css';
+
+import ReactMapboxGl from 'react-mapbox-gl';
+import MapViewModal from './MapViewModal';
+
+type Props = {};
+
+const ResourceMap = (props: Props) => {
+  const { selectedResource } = props;
+  const [modalOpened, setModalOpened] = useState<Boolean>(false);
+
+  useEffect(() => {
+    setModalOpened(true);
+  }, [selectedResource]);
+
+  const Map = ReactMapboxGl({
+    accessToken:
+      'pk.eyJ1IjoiYW5vb2psYWwiLCJhIjoiY2syemtiYjZoMGp1' +
+      'eDNscXQ3azJzajl0bCJ9.FDSFjP1IfSisbm4uvd70vg',
+    interactive: true,
+  });
+
+  return (
+    <div className="map">
+      <Map
+        style="mapbox://styles/mapbox/light-v9"
+        containerStyle={{
+          height: '600px',
+          width: '800px',
+        }}
+        zoom={[15]}
+      />
```

Comment:
If you're interested in trying to have the map not refresh every time, maybe try wrapping this entire component in a `useMemo`
So something like `const mapComponent = useMemo(() => <Map style= .... />, [])` and then `<div className="map">{mapComponent}...</div>`
Maybe might have to wrap `Map` in a memo as well? but if that doesn't work immediately no worries, we can try and figure something out later